lib/cc/services/github_issues.rb

```diff
@@ -55,6 +55,10 @@ def receive_issue
 
 private
 
+  def base_url
+    ENV["CODECLIMATE_GITHUB_API_URL"] || DEFAULT_BASE_URL
```

Comment:
Can this defaulting happen in `CodeClimate::Config` and use `CodeClimate.github_api_url` everywhere?